convex/schema.ts

```diff
@@ -56,11 +56,17 @@ const quests = defineTable({
   steps: v.optional(v.array(v.id("questSteps"))),
   /** Questions related to the quest. */
   faqs: v.optional(v.array(v.id("questFaqs"))),
+  /** Time in ms since epoch when the quest was last updated. */
+  updatedAt: v.optional(v.number()),
```

Comment:
```suggestion
  updatedAt: v.number(),
```

I wonder if we just make this required in order to avoid the fallback "never" state?

Reply:
Followup to "just make it required":

- You'll get a Convex error if you try to make the change because it enforces the schema against the table in your dev environment.
- When this happens I typically just delete the dev data and the schema validates. It's easy to do this from the Convex dashboard.
- For prod, however, we would need to create a migration and backfill data. I haven't actually had to set up migrations yet, since there's been so little data in prod. But now might be a good time. There's a [Migrations Convex component](https://www.convex.dev/components/migrations) which we should set up and get comfortable using before release. This [Intro to Migrations](https://stack.convex.dev/intro-to-migrations) guide from Convex is also helpful. Related issue: #71 

Writing all of this out now... suggested order of operations:

1. Get this merged **as-is** with the field optional
2. Set up migrations using the Convex component
3. Write a migration to backfill the `updatedAt` data for quests in prod
4. Make the `updatedAt` field required and remove conditionals in the frontend

Reply:
Filed here for myself, but maybe it’d be good to add me to the org at some point so I can assign / put in the milestone, etc. https://github.com/namesakefyi/namesake/issues/393